protos/batch.proto

```diff
@@ -17,6 +17,7 @@ syntax = "proto3";
 
 option java_multiple_files = true;
 option java_package = "sawtooth.sdk.protobuf";
+option go_package = "sawtooth_sdk/protobuf";
```

Comment:
The go_package should perhaps include the proto name; for example "sawtooth_sdk/protobuf/batch", since having these all in the same namespace is awkward.